handsontable/src/plugins/autofill/autofill.js

```diff
@@ -124,7 +126,11 @@ export class Autofill extends BasePlugin {
   }
 
   /**
-   * Updates the plugin state. This method is executed when {@link Core#updateSettings} is invoked.
+   * Updates the plugin's state.
+   *
+   * This method is executed when [`updateSettings()`](@/api/core.md#updatesettings) is invoked with any of the following configuration options:
+   *  - `autofill`
+   *  - [`fillHandle`](@/api/options.md#fillhandle)
```

Comment:
In reference to the comment above, this JSDoc could be wrapped in `eslint-disable` + `eslint-enable`, so the entire file would not be disabled.

```
  /* eslint-disable jsdoc/require-description-complete-sentence */
  /**
   * Updates the plugin's state.
   *
   * This method is executed when [`updateSettings()`](@/api/core.md#updatesettings) is invoked with any of the following configuration options:
   *  - `autofill`
   *  - [`fillHandle`](@/api/options.md#fillhandle)
   */
  /* eslint-enable jsdoc/require-description-complete-sentence */
  ```

Reply:
The `complete-sentence` rule is not useful and causes legibility problems. Can we just turn it off for all JSDocs?